docs/configuration.md

```diff
@@ -0,0 +1,473 @@
+## Configuration
+Configuration parameters could be provided upon instantiation of the player instance.
+```js
+var config = {
+  // Configuration here
+};
+var player = playkit.core.loadPlayer(config);
+```
+
+#### Configuration Structure
+```js
+{
+  logLevel: string,
+  type: string,
+  playback: PKPlaybackConfigObject,
+  sources: PKSourcesConfigObject,
+  metadata: PKMetadataConfigObject,
+  plugins: PKPluginsConfigObject
+}
+```
+
+#### Default Configuration Values
```

Comment:
I suspect this will get out of sync very quick, better link to player-config file here

Reply:
@OrenMe  - which link can we use here instead? I agree we should use it and not list here.

Reply:
https://github.com/kaltura/playkit-js/blob/master/src/player-config.json